cloud/blockstore/tests/csi_driver/test.py

```diff
@@ -360,6 +392,7 @@ def test_csi_sanity_nbs_backend(mount_path, volume_access_type):
 
         params_file = Path(os.getcwd()) / "params.yaml"
         params_file.write_text(f"backend: {backend}")
```

Comment:
Let us use yaml.safe_dump here, shall we?

Reply:
Done